packages/backend/src/peripherals/database/migrations/023_tx_count.ts

```diff
@@ -0,0 +1,27 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.createTable('tx_count', function (table) {
+    table.string('project_id').notNullable()
+    table.bigInteger('unix_timestamp').notNullable()
```

Comment:
please let's switch to storing dateTime for dates. This makes working with dates in SQL easier. I did it here: https://github.com/l2beat/l2beat/pull/633/files#diff-7a11975ed53236d7e5a99aa0d987cf7f7fd2bba17c978b2a55e9cc710b0e578eR20